src/actions/tests/sources.spec.js

```diff
@@ -54,7 +54,7 @@ describe("sources", () => {
     expect(source.get("id")).toEqual("foo1");
 
     const locations = getOutOfScopeLocations(getState());
-    expect(locations.length).toEqual(1);
+    expect(locations.length).toEqual(0);
```

Comment:
This tests loads a fake "foo1" source, which is [1] and set a bp at Line 1. Originally it judged this was out of scope but that contradicts what we wanna fix in this bug. In this bug this case in fact should be ruled as in-scope, otherwise the similar case [2] would be out of scope too, so update the expected test result.

[1] "function foo1() {\n  return 5;\n}": https://github.com/devtools-html/debugger.html/blob/9a2977bdb5bbb3a85effd63a315d716d9656fda3/src/actions/tests/helpers/threadClient.js#L71

[2]
```js
let obj = { bar: "bar", foo: function foo1() {  return 5; } };
```

Reply:
No longer have to update sources.spec.js test because it has been modified by PR #4143 to include the column criterion.